### PR TITLE
Instrument the diff comments feature

### PR DESCRIPTION
**Pull request by rubhanazeem**
Instrumenting the feature introduced in #13912

### CODE REVIEW

File: src/api/app/models/comment.rb [hunk 1]
```diff
@@ -13,8 +13,8 @@ class Comment < ApplicationRecord
 
   validate :validate_parent_id
 
-  after_create :create_event
-  after_destroy :delete_parent_if_unused
+  after_create :create_event, :instrument_diff_comments_on_create
+  after_destroy :delete_parent_if_unused, :instrument_diff_comments_on_destroy
```
Comment (hennevogel) [inline src/api/app/models/comment.rb:17]: Why distinguish by state?

Comment (rubhanazeem) [reply src/api/app/models/comment.rb:17]: To get the total number of times the feature has been used. `diff_comment` values are different for both states `(add/remove)`.

**Review by hennevogel — CHANGES_REQUESTED** [review]
We usually instrument the events (like `Event::CommentEvent`) and not some single operation

**Comment by krauselukas** [comment]
@hennevogel @rubhanazeem I took this over, I went the route through the event system. Going to close this PR. Replaced by https://github.com/openSUSE/open-build-service/pull/13969

**Pull request closed without merging**